### ISSUE TITLE
Confusing

### ISSUE THREAD

**Issue by SamuelTheSuperb** [w3x731/survivIoAim#7]
I do not explain how to make it work on google chrome. The ReadMe did not help.

**Comment by w3x731** [comment]
1. [Download](https://github.com/w3x731/survivIoAim/archive/master.zip) this repo and unzip
2. In chrome open link `chrome://extensions`
3. Drag and drop `survivIoAimChromeExtension` folder into chrome

**Comment by SamuelTheSuperb** [comment]
The Zoom works, but not the aimbot. I installed both into chrome.

**Comment by w3x731** [comment]
Yeah, i missed this moment into instructions. Just **uninstall** both extensions and **install only aim** (zoom cheat now not effective, and will be deleted into future). Both extensions dont work correctly at the same time.

**Comment by SamuelTheSuperb** [comment]
thank you, but why is it so laggy?is it just me?

**Comment by SamuelTheSuperb** [comment]
Is it just my computer lagging?

**Comment by w3x731** [comment]
Cant see any lags on my notebook core i5. What do you mean by lags? Shaken weapon in the hands is a normal behaviour. And if you dont have more then 4x zoom, you cant see the full map, only instantly appearing  objects (tees, enemies, etc.) on the screen border.
